cmd/ctr/commands/run/run.go

```diff
@@ -252,7 +253,13 @@ func fullID(ctx context.Context, c containerd.Container) string {
 func buildLabels(cmdLabels, imageLabels map[string]string) map[string]string {
 	labels := make(map[string]string)
 	for k, v := range imageLabels {
-		labels[k] = v
+		if err := clabels.Validate(k, v); err == nil {
```

Comment:
I don't remember why we limit the size of key/value. Is it possible to remove the limitation?

Reply:
We can discuss that in a separate issue / PR for the main branch, and then consider backporting.